specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.ApiManagement/ApiManagement.tsp

```diff
@@ -130,6 +130,70 @@ model ApiManagementGatewayApiRemovedEventData {
   ...ApiGatewayApiBase;
 }
 
+/** Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.CircuitBreaker.Opened event. */
+model ApiManagementCircuitBreakerOpenedEventData {
+  /** Name of the backend for which the circuit has opened. */
+  backendName: string;
+
+  /** Information related to the circuit breaker configured on the backend. */
+  circuitBreaker: ApiManagementCircuitBreakerInfo;
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.CircuitBreaker.Closed event. */
+model ApiManagementCircuitBreakerClosedEventData {
+  /** Name of the backend for which the circuit has closed. */
+  backendName: string;
+
+  /** Information related to the circuit breaker configured on the backend. */
+  circuitBreaker: ApiManagementCircuitBreakerInfo;
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.GatewayTokenNearExpiry event. */
+model ApiManagementGatewayTokenNearExpiryEventData {
+  /** Information related to a given self-hosted gateway deployment. */
+  gatewayInfo: ApiManagementGatewayInfo;
+
+  /** Information related to a an expired gateway token for a self-hosted gateway deployment. */
+  tokenInfo: ApiManagementNearExpiryGatewayTokenInfo;
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.GatewayTokenExpired event. */
+model ApiManagementGatewayTokenExpiredEventData {
+  /** Information related to a given self-hosted gateway deployment. */
+  gatewayInfo: ApiManagementGatewayInfo;
+
+  /** Information related to a an expired gateway token for a self-hosted gateway deployment. */
+  tokenInfo: ApiManagementExpiredGatewayTokenInfo;
+}
+
+/** Information related to the circuit breaker configured on the backend. */
+model ApiManagementCircuitBreakerInfo {
```

Comment:
Rather than Info suffix, can we use "Properties", e.g. ApiManagementCircuitBreakerProperties.